packages/cozy-viewer/src/ViewersByFile/styles.styl

```diff
@@ -28,6 +28,18 @@
 
 // rules for specific viewers below
 
+.viewer-pdfviewer-input-page
+    margin-right     0.25rem
+    text-align       center
+    color            rgb(255, 255, 255)
+    background-color rgb(50, 54, 63)
+    border           2px solid rgba(255, 255, 255, 0.32)
+    border-radius    2px
+    font-size        0.9rem
```

Comment:
Maybe you can explain why it is non dynamic (in the commit message or here)

Reply:
Added to commit 👍